src/gui/folderstatusmodel.cpp

```diff
@@ -1071,14 +1071,10 @@ void FolderStatusModel::slotFolderSyncStateChange(Folder *f)
             if (other != f && other->isSyncRunning())
                 pos += 1;
         }
-        QString message;
-        if (pos <= 0) {
-            message = tr("Waiting...");
-        } else {
-            message = tr("Waiting for %n other folder(s)...", "", pos);
+        if (pos > 0) {
+            pi._overallSyncString = tr("Waiting for %n other folder(s)...", "", pos);
```

Comment:
Unrelated.
We will also display a progress bar in this condition which might look even stranger without text.
![image](https://user-images.githubusercontent.com/200626/146733376-03023d28-2be8-460f-9bc1-e4b4ed62cd6c.png)

Reply:
Hmm? Have you tried with that patch? That looks different in my branch. No bar..

Reply:
You are right the visibility of the bar is not depending on well progress but on overallString.isEmpty()